views/mailinglistwidget.ejs

```diff
@@ -0,0 +1,44 @@
+<ul id="groups" class="group-list">
+	<li id="<%= groups[i].urlName %>">
+    <form action="/mailinglists/<%= groups[i].urlName %>" method="post">
+			<% if (emails.length == 1) { %>
```

Comment:
As far as I can tell, this if block encompasses all the template code on this page. (not closed until line 40) This would explain why nothing renders for a user with multiple accounts